docs/source/01_intro.md

```diff
@@ -1,5 +1,48 @@
 # Introduction
 
+ADAM is a library and command line tool that enables the use of [Apache
+Spark](https://spark.apache.org) to parallelize genomic data analysis across
+cluster/cloud computing environments. ADAM uses a set of schemas to describe
+genomic sequences, reads, variants/genotypes, and features, and can be used
+with data in legacy genomic file formats such as SAM/BAM/CRAM or VCF, as well
```

Comment:
I think it would be useful to link to the supported file types. Someone reading the intro docs may be trying to figure out if ADAM will work for them and linking to the supported file types lets them quickly check if their preferred type is supported.